index.html

```diff
@@ -0,0 +1,101 @@
+    <link rel="stylesheet" href="styles/home.css" type="text/css">
+    <title>GITShop</title>
+</head>
+<body> 
+    <!-- Menu for small devices --> 
+    <div class="menu-content">
+        <div class="user-type">
+            <div class="buyer">
+                <input type="radio" id="buyer" name="user" checked>
+                <label for="buyer">Buyer</label> 
+            </div>
+            <div class="seller"> 
+                <input type="radio" id="seller" name="user"> 
+                <label for="seller">Seller</label>
+            </div>
+        </div> 
+        <div class="categories"> 
+            <h2>Categories</h2>
+            <div class="technology">
+                <img src="images/Technology.png" alt="Technology">
+                <span>Technology</span>
+            </div>
+            <div class="fashion">
+                <img src="images/Fashion.png" alt="Fashion">
+                <span>Fashion</span>
+            </div>
+            <div class="fitness">
+                <img src="images/Fitness.png" alt="Fitness">
+                <span>Fitness</span>
+            </div>
+            <div class="cars">
+                <img src="images/Car.png" alt="Cars">
+                <span>Cars</span>
+            </div>
+            <div class="beauty">
+                <img src="images/Beauty.png" alt="Beauty">
+                <span>Beauty</span>
+            </div>
+        </div> 
+        <div class="sort-by">
+            <h2>Sort By</h2>
+            <div>
+                <input type="radio" id="price-up" name="price" checked>
+                <label for="price-up">Price</label>
+                <img src="images/up arrow.png" alt="up arrow"> 
+            </div>
+            <div> 
+                <input type="radio" id="price-down" name="price"> 
+                <label for="price-down">Price</label>
+                <img src="images/down arrow.png" alt="down arrow">
+            </div>
+        </div> 
+
+        <div class="cart">
+            <img src="images/Shopping-cart.png" alt="Shopping cart"> 
+            <h3>Cart</h3>
+        </div>
+    </div> 
+
+    <!-- Header Section -->
+    <header id="header">
+        <div class="header-content">
+            <div class="hamburger-menu">
+                <div class="hamburger-menu__lines"></div>
+            </div>
+            <h1 class="brand-name"> GITShop</h1>
+            <div class="cart">
+                <img src="images/Shopping-cart.png" alt="Shopping cart"> 
+                <h3>Cart</h3>
+            </div>
+            <div class="search-icon">
+                <img src="images/Search icon.png" alt="search icon">
+            </div>
+        </div> 
+        <div class="search-bar hide-search-bar">
+            <label for="search">Search products by name</label>
+            <div class="search">
+                <img src="images/Search icon.png" alt="search icon">
```

Comment:
you can move the icon to the right and make it submit button for the search bar

Reply:
There is a form with with input of type submit , so Enter will submit (no need for submit button) 
My plan is to show the results as the user start typing ( if I could 🤞)

Reply:
you can :+1: